app.js

```diff
@@ -72,6 +72,50 @@ const lightboxContent = {
   skills: document.querySelector('#works-lightbox-skills'),
 };
 
+// Validations
+
+const emailPattern = /^[a-z0-9.!#$%&'*+/=?^_`{|}~-]+@[a-z0-9](?:[a-z0-9-]{0,61}[a-z0-9])?(?:\.[a-z0-9](?:[a-z0-9-]{0,61}[a-z0-9])?)*$/;
+const namePattern = /^[a-zA-Z ]+$/;
+
+const formContent = document.getElementById('contact-form');
+const nameError = document.getElementById('name-error');
+const emailError = document.getElementById('email-error');
+const textError = document.getElementById('text-error');
+const errorContainer = document.getElementById('error-input');
+
+function comparer(reGex, value) {
+  return reGex.test(value) === false;
+}
+
+formContent.addEventListener('submit', (e) => {
+  const nameInput = formContent.querySelector('#name-input').value;
+  const emailInput = formContent.querySelector('#email').value;
+  const textInput = formContent.querySelector('#text-input').value;
+  if (nameInput != null && emailInput != null && textInput != null) {
+    if (comparer(namePattern, nameInput) || comparer(emailPattern, emailInput) || !textInput) {
+      e.preventDefault();
+      errorContainer.classList.toggle('hidden');
+      if (namePattern.test(nameInput) === false) {
+        nameError.innerText = 'Remove any special symbols or numbers in your name input';
+      } else {
+        nameError.innerText = '';
+      }
+      if (emailPattern.test(emailInput) === false) {
+        emailError.innerText = 'Write your email in lowercase';
+      } else {
```

Comment:
- [ ] I really like how you styled the borders of the input, can you please check if the validations are implemented correctly?  🧐️ Kindly update it so the user can get a more specific error message.
This is the only thing I get when the input is invalid.
![Screenshot from 2022-12-23 20-04-39](https://user-images.githubusercontent.com/17243325/209374678-e14acc0a-359e-444f-ba16-b698f896d53f.png)